pages/common/airmon-ng.md

```diff
@@ -0,0 +1,20 @@
+# airmon-ng
+
+> Activate monitor mode on your wireless network devices.
+> More information: <https://www.aircrack-ng.org/doku.php?id=airmon-ng>.
+
+- List wireless devices and check status:
+
+`sudo airmon-ng`
+
+- Turn on monitor mode for device wlan0:
+
+`sudo airmon-ng start wlan0`
+
+- Kill disturbing processes that use wireless devices:
+
+`sudo airmon-ng check kill`
+
+- Turn off monitor mode for interface wlan0mon:
+
+`sudo airmon-ng stop wlan0mon`
```

Comment:
Missing token syntax?
```suggestion
`sudo airmon-ng stop {{wlan0mon}}`
```

Also, is there any reason why this is `wlan0mon` and not `wlan0` as it was in the previous example above?

Reply:
When set in monitor mode, the interface is renamed to <previous_name>mon